.github/workflows/tests.yml

```diff
@@ -51,8 +51,15 @@ jobs:
         run: |
           docker build -t brew .
```

Comment:
We should probably switch this to using GitHub's native `container:` implementation.

Reply:
IIRC this was tricky to do for some reason. I seem to recall looking into it before unless I've mixed it up with homebrew-portable-ruby CI.

But yes, ideally we do that.

Reply:
Tried it in #923 but I'm getting a weird permission denied error in `setup-ruby`.